pkg/env/proxy/proxy.go

```diff
@@ -82,18 +82,15 @@ func testConn(ctx context.Context, opts []iap.DialOption) error {
 }
 
 func handleClient(ctx context.Context, wizard wizard.Handler, opts []iap.DialOption, conn net.Conn) {
-	wizard.Info(fmt.Sprintf("Client connected: %s", conn.RemoteAddr()))
-	log.Info().Msgf("connected: client %s", conn.RemoteAddr())
+	log.Debug().Msgf("connected: client %s", conn.RemoteAddr())
 
 	tun, err := iap.Dial(ctx, opts...)
 	if err != nil {
-		wizard.Error(fmt.Sprintf("Failed to connect to IAP for client: %s", conn.RemoteAddr()))
```

Comment:
let's keep the error? that's probably still useful to go back to the user

Reply:
OK sure!